selfdrive/ui/qt/widgets/cameraview.cc

```diff
@@ -242,6 +243,10 @@ void CameraViewWidget::paintGL() {
   glPixelStorei(GL_UNPACK_ALIGNMENT, 4);
 }
 
+void CameraViewWidget::vipcDisconnected() {
+  latest_frame = nullptr;
+}
```

Comment:
Need this or we get seg faults if we restart replay/camerad with UI onroad, as latest_frame doesn't get reset, and we may try to call paintGL before we get a frame after connect.